src/autopas/containers/linkedCells/traversals/C18Traversal.h

```diff
@@ -0,0 +1,150 @@
+/**
+ * @file C18Traversal.h
+ * @author nguyen
+ * @date 06.09.2018
+ */
+
+#pragma once
+
+#include "LinkedCellTraversalInterface.h"
+#include "autopas/containers/cellPairTraversals/C18BasedTraversal.h"
+#include "autopas/pairwiseFunctors/CellFunctor.h"
+#include "autopas/utils/WrapOpenMP.h"
+
+namespace autopas {
+
+/**
+ * This class provides the c18 traversal.
+ *
+ * The traversal uses the c18 base step performed on every single cell. Since
+ * these steps overlap a domain coloring with eighteen colors is applied.
+ *
+ * @tparam ParticleCell the type of cells
+ * @tparam PairwiseFunctor The functor that defines the interaction of two particles.
+ * @tparam useSoA
+ * @tparam useNewton3
+ */
+template <class ParticleCell, class PairwiseFunctor, bool useSoA, bool useNewton3>
+class C18Traversal : public C18BasedTraversal<ParticleCell, PairwiseFunctor, useSoA, useNewton3>,
+                     public LinkedCellTraversalInterface<ParticleCell> {
+ public:
+  /**
+   * Constructor of the c18 traversal.
+   * @param dims The dimensions of the cellblock, i.e. the number of cells in x,
+   * y and z direction.
```

Comment:
is this linebreak necessary?

Reply:
https://github.com/AutoPas/AutoPas/pull/148/commits/38e0bd45b346714ffa11f4b7a23c0c9a0b80ec8b